### PR TITLE
Watch instructions

### PR DESCRIPTION
**Pull request by bnmajor**
Add instructions on which variables to set in order to run the watch service separately.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,21 @@ When the ```docker-compose.watch.yml``` file is included in the ```docker-compos
 This message will appear once every minute as the script continues to watch for new timesteps that may have been added. If there is an error message or you do not see the ```Fetching /shots/index.json``` message after the console output has slowed down or stopped, you can get help by creating an [issue](https://github.com/Kitware/eSimMon/issues/new) with the console output.
 
 
+Running the watch service separately
```
Comment (cjh1) [inline README.md:48]: Let call it the monitoring service ( above as well ). We would probably rename some files eventually.

File: README.md [hunk 2]
```diff
@@ -45,6 +45,21 @@ When the ```docker-compose.watch.yml``` file is included in the ```docker-compos
...
+------------------------------------
+In order to run the watch service outside of the stack, the following environment variables will need to be set inside the ```watch.env``` file:
+
+- ```UPLOAD_SITE_URL``` - Should be the url of the site that the data is being exposed on. This is what the script will monitor for new and existing timesteps.
+- ```GIRDER_FOLDER_ID``` - The id of the folder that the data should be uploaded to in the Girder database. This id will need to be updated each time the upload location needs to be changed.
+- ```GIRDER_API_KEY``` - The API key for the Girder instance that the data is being uploaded to. As explained above, this key will automatically be set when the Girder instance is created, but in order to use a different instance.
+- ```GIRDER_API_URL``` - The url for Girder instance that the data will be uploaded to.
```
Comment (cjh1) [inline README.md:55]: url -> URL

File: README.md [hunk 3]
```diff
@@ -45,6 +45,21 @@ When the ```docker-compose.watch.yml``` file is included in the ```docker-compos
 This message will appear once every minute as the script continues to watch for new timesteps that may have been added. If there is an error message or you do not see the ```Fetching /shots/index.json``` message after the console output has slowed down or stopped, you can get help by creating an [issue](https://github.com/Kitware/eSimMon/issues/new) with the console output.
 
 
+Running the watch service separately
+------------------------------------
+In order to run the watch service outside of the stack, the following environment variables will need to be set inside the ```watch.env``` file:
+
+- ```UPLOAD_SITE_URL``` - Should be the url of the site that the data is being exposed on. This is what the script will monitor for new and existing timesteps.
```
Comment (cjh1) [inline README.md:52]: url -> URL

File: README.md [hunk 4]
```diff
@@ -45,6 +45,21 @@ When the ```docker-compose.watch.yml``` file is included in the ```docker-compos
...
+- ```GIRDER_FOLDER_ID``` - The id of the folder that the data should be uploaded to in the Girder database. This id will need to be updated each time the upload location needs to be changed.
```
Comment (cjh1) [inline README.md:53]: id -> ID

File: README.md [hunk 5]
```diff
@@ -45,6 +45,21 @@ When the ```docker-compose.watch.yml``` file is included in the ```docker-compos
...
+- ```GIRDER_API_KEY``` - The API key for the Girder instance that the data is being uploaded to. As explained above, this key will automatically be set when the Girder instance is created, but in order to use a different instance.
+- ```GIRDER_API_URL``` - The url for Girder instance that the data will be uploaded to.
+
+The watch container can then be brought up with docker-compose:
+
+    cd <repo>/devops/docker
+    docker-compose -p esimmon -f docker-compose.watch.yml up
```
Comment (cjh1) [inline README.md:60]: Let add a final sentence indicating what the user should see if all goes well.

File: README.md (hunk 1 above)
Comment (bnmajor) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (bnmajor) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (bnmajor) [reply README.md]: Fixed

File: README.md (hunk 4 above)
Comment (bnmajor) [reply README.md]: Fixed

File: README.md (hunk 5 above)
Comment (bnmajor) [reply README.md:60]: Added in

**Comment by cjh1** [comment]
@bnmajor Perfact thanks

**Pull request merged**